setup.sh

```diff
@@ -1,20 +1,38 @@
 #!/bin/bash
 
+# Set up your .env file BEFORE running this script!!!
+# export all variables from .env
+set -a
+source .env
+set +a
```

Comment:
I'm confused what this does, could you explain it? I think [this](https://stackoverflow.com/a/30969768/7443523) is what you're doing, but it uses `-o`.